client/service.go

```diff
@@ -250,6 +266,10 @@ func (s *Service) UnsubscribeMultiple(topics []string) GenericFuture {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
 
+	for _, v := range topics {
+		s.subscriptions.Remove(v, nil)
```

Comment:
You need to use `Tree.Empty(v)` otherwise the subscription isn't removed. `Remove` will only remove values that macht the second parameter.